src/library/scala/collection/immutable/BitSet.scala

```diff
@@ -45,6 +45,11 @@ sealed abstract class BitSet
     } else this
   }
 
+  override def diff(other: collection.Set[Int]): BitSet = other match {
+    case otherBitset: BitSet => diffBitSet(otherBitset)
+    case _ => super.diff(other)
+  }
+
```

Comment:
Why isn’t it possible to override `diff` in `collection.Bitset`?

Reply:
Because `diff` is not implemented in `collection.Set`.
It is defined in `collection.Set` and implemented in `immutable.Set` and `mutable.Set`. For that reason a bit specific implementation in `collection.BitSet` could not delegate to the default implementation.

Reply:
OK I see. You could probably still implement it in `collection.BitSet` by using an `abstract override` but anyway this problem will be fixed once https://github.com/scala/scala/pull/6782 is merged.